generator/aggregator.go

```diff
@@ -99,7 +78,7 @@ type ServiceExport struct {
 	Exports []string
 }
 
-func CreateAggregateServicesFile(serviceFiles []*gp.FileDescriptorProto, protocTsPath string, statePkg string) (*File, error) {
+func (this *GenericOutputter) CreateAggregateServicesFile(serviceFiles []*gp.FileDescriptorProto, outputType state.OutputTypes, protocTsPath string, statePkg string) (*File, error) {
```

Comment:
What is `outputType` used for? It doesn't look like the parameter is used.

Reply:
ah, an artifact from my earlier investigations of how to get this architecture to work. Nice catch